website/docs/en/community/releases/v1-0.mdx

```diff
@@ -0,0 +1,95 @@
+---
+published_at: 2024-09-10 18:00:00
+---
+
+# Announcing Rsbuild 1.0
+
+> September 10, 2024
+
+![](https://assets.rspack.dev/rsbuild/assets/rsbuild-1-0-banner.png)
+
+We are pleased to announce the release of Rsbuild 1.0!
+
+## Why Rsbuild
+
+For a long time, developers using webpack have been bothered by two major issues: **slow build times and configuration complexity**.
+
+We have used Rust to rewrite webpack into [Rspack](https://github.com/web-infra-dev/rspack), which addresses the slow build issue. However, to maintain compatibility with the webpack ecosystem, Rspack retains webpack's configuration and API, which means it still has some complexity and a learning curve.
+
+### Evolution of the ecosystem
+
+In the early days, there were some excellent tools within the webpack ecosystem, such as Create React App (CRA) and Vue CLI. These tools provided best practices for building React or Vue applications, while hiding the complex webpack configuration. As a result, many React and Vue users used these tools to build applications without having to configure webpack from scratch.
+
+As the ecosystem evolved, full-stack web frameworks such as Next.js, Nuxt, and Remix became popular; Vite was introduced as a lightweight build tool and also gained popularity. However, CRA and Vue CLI gradually stopped getting maintained.
+
+When we look at the npm download numbers for webpack, CRA, and Vue CLI, we find that a large number of projects are still using these tools. For example, webpack has about 25 million weekly downloads, and CRA has nearly 3 million weekly downloads. Many of these projects are CSR applications that do not require the SSR features of full-stack frameworks. Vite seems like a good choice, but after using Vite in our ByteDance projects, we found that the migrating from webpack to Vite comes with high costs and introduces new problems, such as dev and build inconsistency, and slow page refreshes in large applications during development.
```

Comment:
```suggestion
When we look at the npm download numbers for webpack, CRA, and Vue CLI, we find that a large number of projects are still using these tools. For example, webpack has about 25 million weekly downloads, and CRA has nearly 3 million weekly downloads. Many of these projects are CSR applications that do not require the SSR features of full-stack frameworks. Vite seems like a good choice, but after using Vite in our ByteDance projects, we found that the migration from webpack to Vite comes with high costs and introduces new problems, such as dev and build inconsistency, and slow page refreshes in large applications during development.
```

Reply:
How about removing `the`?

![image](https://github.com/user-attachments/assets/9b06b9d1-7fbd-44ec-bcf5-58aca9df910e)

https://www.deepl.com/en/write#en/en-us/Vite%20seems%20like%20a%20good%20choice%2C%20but%20after%20using%20Vite%20in%20our%20ByteDance%20projects%2C%20we%20found%20that%20the%20migration%20from%20webpack%20to%20Vite%20comes%20with%20high%20costs%20and%20introduces%20new%20problems%2C%20such%20as%20dev%20and%20build%20inconsistency%2C%20and%20slow%20page%20refreshes%20in%20large%20applications%20during%20development.

Reply:
Yes that also works !